src/scss/_custom.scss

```diff
@@ -1,3 +1,16 @@
-p {
-  color: red;
+.leftcol {
+  background-color: blue;
+  float: left;
+  clear: right;
+  display: inline-block;
+  width: 30%;
+  margin: 10px;
+}
+
+.rightcol {
+  background-color: green;
+  float: left;
+  display: inline-block;
+  width: 60%;
+  margin: 10px;
```

Comment:
Similarly, a better choice to get the two columns to play nice with one another is to use `padding` inside them and `box-sizing: border-box` as in the "Don't Overthink It Grids" article.